packages/rs-drive-proof-verifier/Cargo.toml

```diff
@@ -18,13 +24,17 @@ dapi-grpc = { path = "../dapi-grpc" }
 drive = { path = "../rs-drive", default-features = false, features = [
   "verify",
 ] }
-dpp = { path = "../rs-dpp", features = ["bls-signatures"], default-features = false }
+dpp = { path = "../rs-dpp", features = [
+  "bls-signatures",
+], default-features = false }
+bincode = { version = "2.0.0-rc.3", features = ["serde"], optional = true }
```

Comment:
why add bincode?

Reply:
There is some mess with serialization.

Not all objects implement PlatformVersionEncode or similar traits, so I use bincode encoding to serialize these objects when dumping messages for Mocks. See packages/rs-sdk/src/mock/requests.rs .

Ideally, all objects should just implement PlatformVersionEncode, so that it will be all much simpler. But that's refactoring I would suggest to do separately.